play-ebean/src/main/java/play/db/ebean/DefaultEbeanConfig.java

```diff
@@ -214,7 +214,7 @@ private static class Classpath {
          * @return a set of types names satisfying the condition
          */
         static Set<String> getTypes(Environment env, String packageName) {
-            return getReflections(env, packageName).getStore().keys(TypeElementsScanner.class.getSimpleName());
+            return getReflections(env, packageName).getStore().getOrDefault(TypeElementsScanner.class.getSimpleName(), Collections.emptyMap()).keySet();
```

Comment:
See https://github.com/ronmamo/reflections/blob/0.9.12/src/main/java/org/reflections/Store.java#L101-L104

Reply:
Now Store is a Map itself: https://github.com/ronmamo/reflections/blob/0.10.2/src/main/java/org/reflections/Store.java